src/constants.ts

```diff
@@ -184,6 +184,8 @@ const Constants = {
         CacheIdentity: 'cacheIdentity',
         AudienceAPI: 'audienceAPI',
         CaptureIntegrationSpecificIds: 'captureIntegrationSpecificIds',
+        CollectClickIdV2Enabled: 'collectClickIdV2Enabled',
+        IntegrationCaptureMode: 'integrationCaptureMode',
```

Comment:
No need to have this in the feature flag constants since that should only be keys coming back from the server.  Move this to where you are using it.

Reply:
I also realized that although these are classified as FeatureFlags, they aren't necessarily ALL feature flags. It is starting to become a catchall for both feature flags AND web input specific settings that come down from the server.

Not a concern for your purposes right now, but I also wanted to note that `collectClickIdV2Enabled` is not what comes from the server. It's '`captureIntegrationSpecificIds.V2'` that you should be looking for, which has values of 'all', 'none', or 'roktonly'
```suggestion
        CaptureIntegrationSpecificIdsV2: 'captureIntegrationSpecificIds.V2',
```